### PR TITLE
chore: jsonfmt storage

### PR DESCRIPTION
**Pull request by nschonni**
Ran `jsonfmt -w "specification/storage/**/*.json"`

### CODE REVIEW

**Comment by nschonni** [comment]
Fixed some model validation issues, but I can move those to a different PR if you want

File: specification/storage/resource-manager/Microsoft.Storage/stable/2017-06-01/examples/OperationsList.json [hunk 1]
```diff
@@ -11,134 +11,119 @@
             "display": {
               "provider": "Microsoft Storage",
               "resource": "Storage Accounts",
-              "operation": "Create/Update Storage Account",
-              "description": "Creates a storage account with the specified parameters or update the properties or tags or adds custom domain for the specified storage account."
```
Comment (tjprescott) [inline specification/storage/resource-manager/Microsoft.Storage/stable/2017-06-01/examples/OperationsList.json]: How/why is this removing this metadata?

**Review by tjprescott — CHANGES_REQUESTED** [review]
This PR is seemingly throwing away several description fields for different methods. That's not a purely formatting change.

File: specification/storage/resource-manager/Microsoft.Storage/stable/2017-06-01/examples/OperationsList.json (hunk 1 above)
Comment (nschonni) [reply specification/storage/resource-manager/Microsoft.Storage/stable/2017-06-01/examples/OperationsList.json]: To fix some model validation errors. I can remove the separate commit and submit it separately if preferred. There is a bug in VS Code with the schemas suggesting that authors add "descriptions" in example files, but the "description" isn't part of the Swagger properties

**Review by tjprescott — APPROVED** [review]

**Comment by tjprescott** [comment]
Can't merge due to failing checks.

**Comment by nschonni** [comment]
@jhendrixMSFT can you help land this?

**Review by jhendrixMSFT — APPROVED** [review]

**Pull request merged**